packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx

```diff
@@ -130,6 +132,28 @@ const TeamPromptOptionsMenu = (props: Props) => {
           openRecurrenceSettingsModal()
         }}
       />
+      <MenuItem
+        key='slack'
+        label={
+          <Link
+            to={`/team/${team.id}/settings/integrations`}
+            target='_blank'
+            rel='noopener noreferrer'
+          >
+            <OptionMenuItem>
+              <SlackSVG />
+              <span className='ml-2'>Configure slack</span>
+              <OpenInNew className='ml-1 text-base text-slate-600' />
```

Comment:
+1 imo it'd be better to push it to the right, looks a little bit cleaner 
<img width="297" alt="Screenshot 2023-09-21 at 12 30 35" src="https://github.com/ParabolInc/parabol/assets/1017620/354dd964-864b-4575-8215-c77b8c9242bd">